### PR TITLE
jahara_league-statistics_count-of-teams

### PR DESCRIPTION
**Pull request by jaharaclark**
Adds functionality for count_of_teams method and calls said method on runner.

### CODE REVIEW

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+require 'CSV'
+
+class StatTracker
+  def self.from_csv(locations)
+    CSV.foreach(locations[:games], headers: true, header_converters: :symbol) do |row|
+      away_team_id = row[:away_team_id].to_i
+    end
+  end
+
+  def self.count_of_teams(locations)
```
Comment (r-stemmle) [inline lib/stat_tracker.rb:10]: This still spikes my interest as far as being a class method (I feel like it should be an instance method)... I would like you to show me how this works as far as scalability/ big picture.

File: lib/stat_tracker.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+require 'CSV'
+
+class StatTracker
+  def self.from_csv(locations)
+    CSV.foreach(locations[:games], headers: true, header_converters: :symbol) do |row|
+      away_team_id = row[:away_team_id].to_i
+    end
+  end
```
Comment (gitkseniya) [inline lib/stat_tracker.rb:8]: Great start but i would like to apply Mike's methodology to the StatTracker

File: runner.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+locations = {
+  games: game_path,
+  teams: team_path,
+  game_teams: game_teams_path
+}
+
+stat_tracker = StatTracker.from_csv(locations)
+
+count_of_teams = StatTracker.count_of_teams(locations)
```
Comment (gitkseniya) [inline runner.rb:15]: We need to keep runner file clean and place methods into corresponding classes

File: runner.rb [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+require 'pry'; binding.pry
```
Comment (gitkseniya) [inline runner.rb:17]: Let's ensure our workspace is clean before submitting PR

**Review by gitkseniya — CHANGES_REQUESTED** [review]
Great start, needs some work!

**Pull request closed without merging**